tests/Core/Components/Switch/FluentSwitchTests.razor

```diff
@@ -0,0 +1,80 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using System.ComponentModel.DataAnnotations
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentSwitchTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentSwitch_Default()
```

Comment:
Can you add a "default" test containing only the component (without parameter): `<FluentSwitch  />`
Mainly yo verify that non extra html attributes are present.

Reply:
Simpler default test : `<FluentSwitch />`